sentences/de/homeassistant_HassCancelTimer.yaml

```
@@ -0,0 +1,12 @@
+language: de
+intents:
+  HassCancelTimer:
+    data:
+      - sentences:
+          - "<timer_cancel>[ den| meinen] Timer"
+          - "<timer_cancel>[ den| meinen] <timer_start> Timer"
+          - "<timer_cancel>[ den| meinen] Timer für <timer_start>"
+          - "<timer_cancel>[ den| meinen] {area} Timer"
```

```suggestion
          - "<timer_cancel>[ den| meinen] <area> Timer"
```
were these `{` brackets intentionally?

---

Yes. For DE the <area> expansion rule contains prepositions and articles ("in der küche", "im wohnzimmer"). This sentence definition should only accept the accusative ("den") as in "stoppe _den_ wohnzimmer timer" which is why I directly used the area slot rather than the expansion rule.